data-plane/config/source/100-config-kafka-source-data-plane.yaml

```diff
@@ -157,4 +157,4 @@ data:
     # ssl.trustmanager.algorithm
   config-kafka-source-webclient.properties: |
     idleTimeout=10000
-    # maxPoolSize=50
+    maxPoolSize=50
```

Comment:
wouldn't this be overridden by https://github.com/knative-sandbox/eventing-kafka-broker/blob/21eba1eec533f81186a8e5eb4ae93325803d6221/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java#L341-L346?